fhirpath/internal/parser/visitor.go

```diff
@@ -443,6 +444,12 @@ func (v *FHIRPathVisitor) VisitExternalConstant(ctx *grammar.ExternalConstantCon
 // root of the expression. If so, it will return a TypeExpression. Otherwise, it returns a FieldExpression.
 func (v *FHIRPathVisitor) VisitMemberInvocation(ctx *grammar.MemberInvocationContext) interface{} {
 	identifier := ctx.GetText()
+	if ctx.Identifier().DELIMITEDIDENTIFIER() != nil {
+		if len(identifier) < 2 || identifier[0] != '`' || identifier[len(identifier)-1] != '`' {
+			return &VisitResult{nil, fmt.Errorf("%w: %s", errInvalidDelimitedIdentifier, identifier)}
+		}
+		identifier = identifier[1 : len(identifier)-1]
```

Comment:
We can use `strings.TrimPrefix` and `strings.TrimSuffix` to make this more concise


```suggestion
		identifier = strings.TrimPrefix(identifier, '`')
		identifier = strings.TrimSuffix(identifier, '`')
```

Reply:
Done